Controller/TutorialController.php

```diff
@@ -103,4 +103,31 @@ public function collectionsAction()
             'form' => $form->createView(),
         );
     }
+
+    /**
+     * @Route("/tutorial/send-mail")
+     * @Template()
+     * @return array
+     */
+    public function sendMailAction()
+    {
+        // get the message factory so we can create messages
+        $messageFactory = $this->get('framework.message_factory');
+
+        // create a simple message
+        $message = $messageFactory->createHtmlMessage(
+            'the subject',
+            '<p>foo bar</p>'
+        );
+
+        // set some extra properties, just like you would do with a normal \Swift_Message
+        $message->setTo(
+            $this->getParameter('mailer_default_to_email')
+        );
+
+        // send it
+        $this->get('mailer')->send($message);
+
+        return array();
```

Comment:
The message passed, is wrapped in a template.
It looks like: 
![screen shot 2016-03-11 at 14 43 51](https://cloud.githubusercontent.com/assets/250042/13703643/c609afb0-e797-11e5-94eb-622cbe7aacc9.png)